api/pb/api.proto

```diff
@@ -195,7 +195,7 @@ service API {
     rpc GetToken(stream GetTokenRequest) returns (stream GetTokenReply) {}
     rpc NewDB(NewDBRequest) returns (NewDBReply) {}
     rpc NewDBFromAddr(NewDBFromAddrRequest) returns (NewDBReply) {}
-    rpc GetDBInfo(GetDBInfoRequest) returns (GetDBInfoReply) {}
+    rpc GetInviteInfo(GetInviteInfoRequest) returns (GetInviteInfoReply) {}
```

Comment:
This change will require a tiny client tweak downstream cc @carsonfarmer and @andrewxhill